Correct Typo in Azure Blobs Description

qnal opened the pull request:
A massively scalable object store for text and binary data (instead of "binary day")

azurecla commented:
Hi __@Taojunshen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-jutao). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

qnal committed:
Correct Typo in Azure Blobs Description

A massively scalable object store for text and binary data (instead of "binary day")
